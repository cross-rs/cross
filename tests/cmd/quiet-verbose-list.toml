bin.name = "cross"
args = ["--quiet", "--verbose", "--list"]
