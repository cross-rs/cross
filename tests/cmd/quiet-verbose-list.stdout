+ cargo --quiet --verbose --list
Cross Commands:
...
Host Commands:
...
